src/component/table/Table-test.js

```diff
@@ -0,0 +1,72 @@
+import test from 'ava';
```

Comment:
The location of this file is also a little inconsistent. Should we stick to keeping tests in `__test__` folders?